tests/context.py

```diff
@@ -58,4 +59,4 @@ def get_cluster_name(self):
 
 
 def generate_elastic_url(cluster_name):
-    return "http://elastic-{0}.spacemesh.io".format(cluster_name.split('-')[1])
\ No newline at end of file
+    return "http://elastic-{0}.spacemesh.io".format(cluster_name.split('-')[1])
```

Comment:
make the string a const ?

Reply:
This is not part of the PR it's Itzik code but I did change it though it's not enough.. we shouldn't parse in the middle of the code like that

Reply:
fixed in 5c6b63aabad9a2849d5c4dcc7755c9ab121a0579